README.md

```diff
@@ -1,2 +1,4 @@
 # pumpkin-seeds
 Form of squash with seeds inside.
+
+https://en.wikipedia.org/wiki/Pepita
```

Comment:
Kaelig provided a definition for this edit; this is very important to our marriage.